recipes/openimageio/all/conanfile.py

```diff
@@ -120,7 +120,7 @@ def requirements(self):
         if self.options.with_raw:
             self.requires("libraw/0.21.2")
         if self.options.with_openjpeg:
-            self.requires("openjpeg/2.5.0")
+            self.requires("openjpeg/2.5.2")
```

Comment:
This is required to solve a version conflict between this recipe and ffmpeg

Reply:
Thanks for the bump!